### ISSUE TITLE
Sleep Times Break the script (Error Fixed)

### ISSUE THREAD

**Issue by nyxtra404** [adr1an-debug/snapchat-snapscore-farm#1]
Basically, the waiting times are very short and can break the program, which has happened to me. I have simply changed the times to a value from a minimum of 300 to a maximum of 500, and it goes much better without breaking.

**Comment by adr1anhz** [comment]
Good point. It's mostly about individual pc, for me 300ms not breaking up and its been running for days

**Comment by nyxtra404** [comment]
The thing was that by having such short waiting times, the program would break and instead of sending the message it would just select another person, and you would even lose opportunities to send photos because of the time. So my recommendation is to lower it to a value between 300 and 500.
